cli/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [0.1.0-pre.5]
+
+### Features
+
+- Adds a hint when `init` is run in an initialized project
+
+### Fixes
+
+- Fixes the CLI header being printed when printing completions
+
+### Refactoring
+
+- Cleans up the CLI `main.rs` file
```

Comment:
I wonder if we need these in the change log?

Reply:
IMHO the change log should only include changes of interest to the user

Reply:
For now I'll merge it with the change since there's already other similar lines, we can clean it up seperately